beaconChain/tests/helpers/stateTransitionHelpers.test.ts

```diff
@@ -323,3 +324,59 @@ describe("isActiveValidator", () => {
     assert.isFalse(result);
   });
 });
+
+describe("getForkVersion", () => {
+  const fork: Fork = {
+    epoch: new BN(12),
+    previousVersion: new BN(4),
+    currentVersion: new BN(5)
+  };
+
+  const four: uint64 = new BN(4);
+  const five: uint64 = new BN(5);
+
+  it("fork version should be 4", () => {
+    const result = getForkVersion(fork, new BN(8));
+    assert(result.eq(four));
+  });
+
+  it("fork version should be 5", () => {
+    const result = getForkVersion(fork, new BN(13));
+    assert(result.eq(five));
+  });
+
+  it("fork version should be 5", () => {
+    const result = getForkVersion(fork, new BN(12));
+    assert(result.eq(five));
+  });
+});
+
+describe("getDomain", () => {
+  const fork: Fork = {
+    epoch: new BN(12),
+    previousVersion: new BN(4),
+    currentVersion: new BN(5)
+  };
+
+  const constant: uint64 = new BN(2 ** 32);
+  const four: uint64 = new BN(4);
+  const five: uint64 = new BN(5);
+
+  it("domain version should be 4", () => {
```

Comment:
I think we can remove the word "version" here and below.
And domain isn't 4 and 5.
maybe something like the above comment:
"epoch before fork epoch should result in domain === previous fork version * 2**32 + domain type" etc.

Reply:
Got lazy on these .... lol good catch!